### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by innovate-invent**
This merges the jmespath.tests repo such that compliance tests can be included with a JEP

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,86 +1,88 @@
...
-### Go
-
-```go
-```
-
-### Lua
-
-```lua
-```
+<!--
```
Comment (springcomp) [inline .github/pull_request_template.md:1]: Beware of line endings. Seems this commit changed the whole file.

File: .github/workflows/test_validate.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v3
+        with:
+          cache: pip
+          cache-dependency-path: bin/requirements.txt
+      - name: Validate tests
+        run: "ls grammar/*.json functions/*.json | xargs -I {} jsonschema test_schema.json -i {}"
```
Comment (springcomp) [inline .github/workflows/test_validate.yaml]: Does this run without error even if the `functions` folder does not exist?
Did you intend to have this PR be a follow up of #52 ?

Comment (innovate-invent) [reply .github/workflows/test_validate.yaml]: Yes, it works even if the folder does not exist.
This is in preparation of #52

File: .github/pull_request_template.md (hunk 1 above)
Comment (innovate-invent) [reply .github/pull_request_template.md:1]: I hadn't realised this was CRLF, are we wanting to standardise on LF?

Comment (springcomp) [reply .github/pull_request_template.md:1]: I guess we should, yes. That's what the majority of users would expect, coming from mostly a Linux background.

Comment (innovate-invent) [reply .github/pull_request_template.md:1]: .gitattributes added

**Review by springcomp — APPROVED** [review]

**Pull request merged**